libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/types.ts

```diff
@@ -190,6 +191,7 @@ export interface IAffectedSharedObject {
 export interface IShareDialogBaseProps {
     sharedObject: IAffectedSharedObject;
     currentUserRef: ObjRef;
+    currentUserPermissions: CurrentUserPermissions;
```

Comment:
Isn't it breaking change when we add a new mandatory property to a public interface?

Reply:
Its ok ... this type is public but mark as internal ... this mean DO not use or expect some possible changes